server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/BasicServlet.java

```diff
@@ -264,4 +264,12 @@ protected static void banner(StringBuilder sb, String klass, String text) {
     sb.append("<br />\n<h2 class='").append(klass).append("'>").append(text).append("</h2>\n");
   }
 
+  /**
+   * Creates a {@link Cookie} with the given name and value, also setting the HttpOnly attribute on the cookie.
+   */
+  protected static Cookie createCookie(String name, String value) {
+    Cookie c = new Cookie(name, value);
+    c.setHttpOnly(true);
```

Comment:
> I can double-check, but I'm not aware of that in 1.x. Not sure how the table sorting stuff works.

So I'm surprised, but this actually just doesn't break anything. Sorting on tables still works. Returning to a page after leaving retains the sort column+order.